### PR TITLE
[Search] Change Azure.Search.Documents back to targeting preview version

### PR DESCRIPTION
**Pull request by ShivangiReja**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/search/Azure.Search.Documents/README.md [hunk 1]
```diff
@@ -171,19 +174,17 @@ For more details about Dependency Injection in ASP.NET Core apps, see [Dependenc
 
 ## Key concepts
 
-An Azure AI Search service contains one or more indexes that provide
+An Azure Cognitive Search service contains one or more indexes that provide
```
Comment (pallavit) [inline sdk/search/Azure.Search.Documents/README.md:177]: Should we not take these changes as they are part of the rebranding?

Comment (ShivangiReja) [reply sdk/search/Azure.Search.Documents/README.md]: Updated!

**Comment by ShivangiReja** [comment]
@pallavit I'll create a seperate PR for all the changes we did for GA.

**Comment by pallavit** [comment]
@ShivangiReja looks like we made some breaking changes in the toolchain so it may be easier to try to overlay Octpreview release on top of GA rather than the other way round.

**Comment by ShivangiReja** [comment]
@pallavit - I've not added any breaking changes. Looks like all the PRs in the azure-sdk-for-net repository are currently failing.
 
Here are the links to the recent PRs for your reference:
[https://github.com/Azure/azure-sdk-for-net/pull/40958](https://github.com/Azure/azure-sdk-for-net/pull/40958)
[https://github.com/Azure/azure-sdk-for-net/pull/40957](https://github.com/Azure/azure-sdk-for-net/pull/40957)

**Review by pallavit — APPROVED** [review]

**Pull request merged**